sdv/metadata/single_table.py

```diff
@@ -1069,6 +1082,7 @@ def load_from_json(cls, filepath):
                 'class and version.'
             )
 
+        cls._updated = False
```

Comment:
why not make the default false?

Reply:
Yes, done in [18ee971](https://github.com/sdv-dev/SDV/pull/1786/commits/18ee9712445700f04cc8bb51490ebf48a1304237)